src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+let total = 0;
+let average = 0;
+
+const numbers = [...document.querySelectorAll('span.population')];
```

Comment:
Ensure that the elements with the class 'population' exist in the DOM. If not, this line will result in an empty array, and subsequent calculations will be incorrect.